<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolbarPage</class>
 <widget class="QWidget" name="ToolbarPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="lockToolbar">
     <property name="text">
      <string>Lock the toolbar position</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="toolbarVisible">
     <property name="text">
      <string>Toolbar visible</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Position</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QRadioButton" name="topRadioButtom">
          <property name="text">
           <string>Top</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="bottomRadioButtom">
          <property name="text">
           <string>Bottom</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rightRadioButtom">
          <property name="text">
           <string>Right</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="leftRadioButtom">
          <property name="text">
           <string>Left</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Buttom style</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_10">
        <item>
         <widget class="QRadioButton" name="onlyIconsRadioButtom">
          <property name="text">
           <string>Only icons</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="onlyTextRadioButtom">
          <property name="text">
           <string>Only text</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tbiRadioButtom">
          <property name="text">
           <string>Text beside icons</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="tuiRadioButtom">
          <property name="text">
           <string>Text under icons</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="fsRadioButtom">
          <property name="text">
           <string>Follow style</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>441</width>
       <height>48</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>toolbarVisible</sender>
   <signal>toggled(bool)</signal>
   <receiver>lockToolbar</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>34</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>10</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
